main.go

```diff
@@ -101,7 +105,11 @@ func main() {
 		os.Exit(1)
 	}
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+	cfg := ctrl.GetConfigOrDie()
+	cfg.QPS = float32(adapterClientRequestQPS)
```

Comment:
after applied the change above, this will be 
```suggestion
	cfg.QPS =adapterClientRequestQPS
```